app/views/dashboards/new_user.html.erb

```diff
@@ -0,0 +1,71 @@
+<div style="width: 500px; margin: 0 auto;">
+  <div class="card m-3">
+    <div class="card-header">
+      <h5>Add a user </h5>
+    </div>
+    <div class="card-body">
+      <%= form_with(model: @user, url: dashboard_add_user_path) do |form| %>
+        <%#= render partial: "shared/form_errors", locals: { object: form.object } %>
```

Comment:
remove comments, if not needed.

Reply:
Bhaiya, it used partial view for minimizing the single file code.
Thank you!